addons/hide-backpack/userscript.css

```diff
@@ -0,0 +1,16 @@
+[class^="backpack_backpack-container"] {
+  display: none;
+}
+
+[class^="button-group_button-group_"] {
```

Comment:
This rule should be in a separate CSS file that's only injected if "show small backpack icon" is enabled.

```suggestion
[class^="paint-editor_zoom-controls_"] {
```
`button-group_button-group_` will select every button group in the paint editor, including undo/redo:
![image](https://user-images.githubusercontent.com/51849865/189313126-010cccb7-e049-4e51-bba6-550319baf007.png)